src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/asides/ts.ui.SideBarSpirit.js

```diff
@@ -161,7 +161,32 @@ ts.ui.SideBarSpirit = (function using(chained, Type, Client, GuiObject, Colors)
 				ts.ui.removeBreakPointListener(this);
 			},
 
-			// Private .................................................................
+			// Privileged ............................................................
+
+			/**
+			 * Show the SideBar (now that it's hidden in mobile view).
+			 * TODO: Call `DoorManPlugin.didopen() when transition is done!
+			 */
+			$onopen: function() {
+				this.css.add('ts-will-open');
```

Comment:
It's only used in this file, though, but OK:  found another constant `ts.ui.CLASS_OPENING` and used that instead.